src/api/app/controllers/webui/users/notifications_controller.rb

```diff
@@ -74,21 +63,53 @@ def show_more(notifications)
 
   def fetch_notifications
     notifications = policy_scope(Notification).for_web.includes(notifiable: [{ commentable: [{ comments: :user }, :project, :bs_request_actions] }, :bs_request_actions, :reviews])
-    notifications_finder = NotificationsFinder.new(notifications)
 
-    if params[:project]
-      notifications_finder.for_project_name(params[:project])
-    elsif params[:group]
-      notifications_finder.for_group_title(params[:group])
+    relations = notifications
```

Comment:
What is "relations" supposed to be?

Reply:
`relations` is supposed to be a big query containing all the filters the user selected in the notifications page.

Reply:
I don't think it means what you think it means.

https://www.dictionary.com/browse/relation

And why not name the variable like that from the start then?

Reply:
I do not understand. What do you mean "from the start"? Are you talking about `notifications`?

Reply:
Yes, we assign the output of `policy_scope` to `notifications` to assign that to `relations` to overwrite it again based on some parameters. This is all very complicated for no real reason right?

Reply:
It is true we assign `relations = notifications` but we could use `notifications` directly.
It's a leftover from the work in progress. Thanks for catching it